sdk/python/feast/usage.py

```diff
@@ -89,14 +120,33 @@ def log(self, function_name: str):
                 "os": sys.platform,
                 "is_test": self._is_test,
             }
-            try:
-                requests.post(USAGE_ENDPOINT, json=json)
-            except Exception as e:
-                if self._is_test:
-                    raise e
-                else:
-                    pass
-            return
+            asyncio.run(self._send_usage_request(json))
+
+    def should_log_for_get_online_features_event(self, event_name: str):
+        if event_name not in self._usage_counter:
+            self._usage_counter[event_name] = 0
+        self._usage_counter[event_name] += 1
+        if self._usage_counter[event_name] % 10000 != 2:
+            return False
+        self._usage_counter[event_name] = 2  # avoid overflow
+        return True
+
+    def log_event(self, event: UsageEvent):
+        self.check_env_and_configure()
+        if self._usage_enabled and self.usage_id:
+            event_name = str(event)
+            if event == UsageEvent.GET_ONLINE_FEATURES_WITH_ODFV:
+                if not self.should_log_for_get_online_features_event(event_name):
+                    return
```

Comment:
it collapse into `if event == ... and not self.should_log_for_get_online_features_event: return`

Reply:
Done